docs/source/getting-started/quickstart/template-root/daml/Setup.daml

```diff
@@ -0,0 +1,47 @@
+-- Copyright (c) 2020 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE ApplicativeDo #-}
```

Comment:
Since each submit only calls one command you don’t actually need this here which might be slightly less scary.

Reply:
Thanks. Fixed in eec86fb.